site/content/contact/_index.md

```diff
@@ -8,7 +8,7 @@ We would be happy to receive your feedback about the software available, what is
 
 ## Discord
 
-You can reach us in our [Discord Server](https://discord.gg/AMuVCRZEvG) and we are also in the [Funtoo Discord Server](https://discord.gg/BNUSpUU).
+You can reach us in our [Discord Server](https://discord.gg/AMuVCRZEvG). The Funtoo Discord server will be missed.
```

Comment:
I think that you can just drop the note about Funtoo Discord no more available

Reply:
At first I thought you meant to undo my change, but you meant for me to remove it like so, right?

Reply:
yeah, exactly. I leave the honor of this change to you considering that is part of this PR.